src/main/java/model/ModelGraph.java

```diff
@@ -69,8 +73,8 @@ public Vertex insertVertex(String id, VertexType vertexType, Point3d coordinates
         return Optional.empty();
     }
 
-    public InteriorNode insertInterior(String id, Vertex v1, Vertex v2, Vertex v3) {
-        InteriorNode interiorNode = new InteriorNode(this, id, v1, v2, v3);
+    public InteriorNode insertInterior(String id, Vertex v1, Vertex v2, Vertex v3, Vertex... associatedNodes) {
+        InteriorNode interiorNode = new InteriorNode(this, id, v1, v2, v3, associatedNodes);
```

Comment:
Make another method, don't change existing, because it is probably used somewhere else.

Reply:
Varargs nie zmieniają bezpośrednio API bo bez ich podania znajdzie się tam pusta lista. Kod gdzie została już użyta dalej się będzie kompilował.